.github/workflows/build.yml

```diff
@@ -318,6 +318,7 @@ jobs:
       - name: Build
         uses: MansaGroup/nrwl-nx-action@v2
         with:
+          # build-multi assumes all required resources are present
           targets: "build-multi"
```

Comment:
Maybe we could specify dependent files with nx somehow?

Reply:
For caching specifically, yes, but I'm not sure it would be super useful since we only use this target in the GH actions.